packages/oui-form-actions/README.md

```diff
@@ -0,0 +1,48 @@
+# Form Actions
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-form-actions
+  on-submit="$ctrl.submit()"
+  on-cancel="$ctrl.cancel()">
+</oui-form-actions>
+```
+
+## API
+
+| Attribute     | Type     | Binding | One-time Binding | Values                 | Default   | Description                      |
+| ----          | ----     | ----    | ----             | ----                   | ----      | ----                             |
+| on-submit     | function | &       |                  |                        |           | submit form action               |
```

Comment:
I think  "handler triggered on-submit" could be more standard.  Same for "on-cancel"

Reply:
oki submit handler & cancel handler to be in adequation with oui-button documentation